etna/models/sklearn.py

```diff
@@ -137,6 +137,7 @@ def __init__(self, regressor: RegressorMixin):
             sklearn model for regression
         """
         super().__init__(base_model=_SklearnAdapter(regressor=regressor))
+        self.regressor = regressor
```

Comment:
I think we should add some regression test, that checks that the problem is fixed. Add a new test case into `tests/test_core/test_to_dict.py/test_to_dict_models` with `SklearnPerSegmentModel` with some sklearn model in `regressor`, e.g. `RandomForest`.

Reply:
Added two models SklearnPerSegmentModel and SklearnMultiSegmentModel with sklearn RandomForestRegressor default params